Remotion/Web/Development.WebTesting/CoypuElementScopeFillInWithAndSendKeysExtensions.cs

```diff
@@ -105,11 +106,26 @@ private static void SetValueUsingSendKeys ([NotNull] this ElementScope scope, [N
       value = value.Replace("\r", "");
 
       var clearTextBoxWithoutTriggeringPostBack = Keys.Control + "a" + Keys.Control + Keys.Delete;
-      value = clearTextBoxWithoutTriggeringPostBack + value;
 
-      s_log.DebugFormat("FillInWith without triggering PostBack on clear: '{0}'.", value);
-
-      scope.SendKeys(value);
+      if (scope.Browser.IsFirefox())
+      {
+        //GeckoDriver cannot handle inputting the value after receiving the delete command, IF both were sent in the same SendKeys.
+        //This occurs after Firefox v99 and would lead to an empty field.
+        s_log.DebugFormat("FillInWith without triggering PostBack on clear: '{0}' followed by '{1}'.", clearTextBoxWithoutTriggeringPostBack, value);
+
+        scope.SendKeys(clearTextBoxWithoutTriggeringPostBack);
+        scope.SendKeys(value);
+      }
+      else
+      {
+        //The firefox code should work for other browsers as well.
+        //BUT due to previous problems concerning insertion of values, we leave the chromium based implementation alone.
```

Comment:
```suggestion
        // The Firefox code has been updated recently (March 2023) and should work for other browsers as well.
        // However, due to previous problems concerning the insertion of values, we leave the Chromium-based implementation unchanged to avoid the risk of regressions.
```